src/Core/tests/UnitTests/Layouts/GridLayoutManagerMinMaxTests.cs

```diff
@@ -0,0 +1,808 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Globalization;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Layouts;
+using NSubstitute;
+using Xunit;
+using static Microsoft.Maui.UnitTests.Layouts.LayoutTestHelpers;
+using static Microsoft.Maui.UnitTests.Layouts.GridLayoutManagerTests;
+
+namespace Microsoft.Maui.UnitTests.Layouts
+{
+    [Category(TestCategory.Layout)]
+    public class GridLayoutManagerMinMaxTests
```

Comment:
Did the robot dupe all these tests? Looks like there's two versions of GridLayoutMangerMinMaxTests.

Reply:
Having duplicate tests means you really make sure it's good.